### PR TITLE
[15.0][MIG] stock_picking_product_assortment_availability_inline: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT40157

ping @chienandalu @carlosdauden

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

**Review by kluna1998 — APPROVED** [review]
Functional review done 👍🏽

**Comment by rousseldenis** [comment]
/ocabot migration stock_picking_product_assortment_availability_inline

File: setup/stock_picking_product_assortment_availability_inline/setup.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import setuptools
```
Comment (rousseldenis) [inline setup/stock_picking_product_assortment_availability_inline/setup.py:1]: @sergio-teruel Don't forget to put such changes in pre-commit commit

**Comment by pedrobaeza** [comment]
/ocabot migration stock_picking_product_assortment_availability_inline
/ocabot merge nobump

**Pull request merged**